CHANGELOG.md

```diff
@@ -1194,7 +1194,7 @@ replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8
 ### Bug Fixes
 
 * Implement dragonberry security patch.
-    * For applying the patch please refer to the [RELEASE NOTES](./RELEASE_NOTES.md)
+    * For applying the patch please refer to the [RELEASE PROCESS](./RELEASE_PROCESS.md)
```

Comment:
It is refer to the release notes. Maybe let's point GitHub release instead. This isn't about the release process.